README.md

```diff
@@ -2688,10 +2704,10 @@ Other Style Guides
     // good
     $('#items')
       .find('.selected')
-        .highlight()
-        .end()
+      .highlight()
+      .end()
       .find('.open')
-        .updateCount();
+      .updateCount();
```

Comment:
Better to place dots under each other

Reply:
not in this case, because in jquery, `.end()` is like "stepping out"